builder/bundles.go

```diff
@@ -346,7 +352,18 @@ func resolvePackages(numWorkers int, set bundleSet, packagerCmd []string, emptyD
 			for _, pkgs := range fullRpm {
 				// Add packages to bundle's AllPackages
 				for _, pkg := range pkgs {
-					name := pkg.name + "-" + pkg.version + "." + pkg.arch + ".rpm"
+					var name string
+					name = pkg.name + "-" + pkg.version + "." + pkg.arch + ".rpm"
+					if pkg.repo == "local" {
+						if _, err := os.Stat(filepath.Join(localPath, name)); os.IsNotExist(err) {
```

Comment:
Can we have this block plus the query string above on a dedicated function? It will make it easier to read the code.